src/NServiceBus.Core/Serialization/SerializationFeature.cs

```diff
@@ -30,10 +30,16 @@ protected internal sealed override void Setup(FeatureConfigurationContext contex
 
             var defaultSerializer = CreateMessageSerializer(defaultSerializerAndDefinition, mapper, settings);
 
+            var additionalDeserializerDefinitions = context.Settings.GetAdditionalSerializers();
```

Comment:
I'm wondering a little bit why we're not using dictionaries instead of tuples

Reply:
Me to :) 

I don't want to change this now. Think the entire serialization thing should be refactored into a component instead going forwards